code/game/dna/mutations/mutation_powers.dm

```diff
@@ -445,7 +445,19 @@
 			doHeal(user)
 	else
 		user.visible_message("<span class='danger'>[user] eats \the [the_item].</span>")
-		playsound(user.loc, 'sound/items/eatfood.ogg', 50, 0)
+		playsound(user.loc, 'sound/items/eatfood.ogg', 50, FALSE)
+		if(ismob(the_item.loc) && isitem(the_item))
+			var/obj/item/eaten = the_item
+			var/mob/the_owner = the_item.loc
+			the_owner.unEquip(eaten, TRUE, TRUE)
```

Comment:
Shouldn't this rather check if it's unequippable even?

Reply:
Done